assets/js/components/notifications/SetupSuccessBannerNotification.js

```diff
@@ -194,26 +199,29 @@ function SetupSuccessBannerNotification() {
 				}
 			}
 
-			const anchor = {
-				link: '',
-				label: '',
-			};
-
 			if ( 'pagespeed-insights' === slug ) {
-				anchor.link = '#speed';
-				anchor.label = __(
-					'Jump to the bottom of the dashboard to see how fast your home page is',
-					'google-site-kit'
-				);
-			}
+				const anchorLink = '#speed';
+				const onJumpLinkClick = ( e ) => {
```

Comment:
Another `e` here that would be great to rename:

```suggestion
				const onJumpLinkClick = ( event ) => {
```